lips/lip-3.md

```diff
@@ -8,52 +8,51 @@ created: 06/09/2020
 
-The goal of LUM is to provide the Association with means to exercise its authority to reconfigure the blockchain and to effectively remediate any plausible operational incident while maintaining the trust gained through decentralized governance and distributed systems.
+The goal of LUM is to provide the means to effectively and transparently update the software in order to meet changing requirements as well as remediate any plausible operational incident.
+
 We first present all building blocks and the reasoning, followed by concrete examples.
 
 # Building blocks
-There’s two types of upgrades in the system - (i) updates to the Libra protocol software and (ii) changes to on-chain state. We discuss them correspondingly.
+There are two types of upgrades in the system - (i) [updates to the Libra protocol software](#software-upgrade) and (ii) [changes to on-chain state](#on-chain-state-upgrade). We discuss them correspondingly.
+
+All building blocks may be used for "business-as-usual" upgrades or to address urgent incidents.
 
 ## Software upgrade
 Specification changes go through the LIP process and non-specification changes (e.g. bug fixes and performance improvements) do not need to initiate the LIP process. All software changes are submitted through standard Github reviews and proceed through the code review process before being merged to master.
 
-Periodically, LN needs to release a new version of software and coordinate the release plan and timeline with all the validator operators. Validator operators are responsible for deploying the software release as per instructions.
+In order to release a new version of software, Libra Networks (LN) needs to coordinate the release plan and timeline with all the validator operators. Validator operators are responsible for deploying the software release as per the provided instructions.
 
-Some specification changes may require all validators to transition at the same height, which is accompanied with an on-chain state value update (described in the next section).
+Some changes may require all validators to transition at the same height, which is accompanied with an [on-chain state update](#on-chain-state-upgrade).
 
 ## On-chain state upgrade
-On-chain state can only be updated via transactions. We propose three ways for LN to execute critical updates, ordered by most frequently expected to least frequently expected: normal multi-sig transactions, multi-sig WriteSet transactions, and genesis transactions.
+On-chain state can only be updated via transactions. We propose three ways to execute critical updates, ordered by most frequent to least frequent: normal multi-sig transactions, multi-sig WriteSet transactions, and genesis transactions.
```

Comment:
why "critical updates"? these could either be "business as usual" or urgent. How about:

.....three ways [to effectuate changes to the LPN by directly mutating on-chain state].